### PR TITLE
hillel- added blue scoring positions

### PR DESCRIPTION
**Pull request by hillelv**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/commands/swerve/MoveToClimbPosition.java [hunk 1]
```diff
@@ -2,10 +2,12 @@
...
 import edu.greenblitz.robotName.Field;
 import edu.greenblitz.robotName.commands.auto.MoveToPosition;
+import edu.greenblitz.robotName.utils.FMSUtils;
+import edu.wpi.first.wpilibj.DriverStation;
 
 public class MoveToClimbPosition extends MoveToPosition {
 
     public MoveToClimbPosition() {
-        super(Field.ScoringPositions.CLIMB_POSITION);
+        super(FMSUtils.getAlliance() == DriverStation.Alliance.Blue ? Field.ScoringPositions.RED_CLIMB_POSITION : Field.ScoringPositions.BLUE_CLIMB_POSITION);
```
Comment (tnrz135) [inline src/main/java/edu/greenblitz/robotName/commands/swerve/MoveToClimbPosition.java:11]: If we are the blue alliance, why do we use the red climb position?
Also, I believe there is a function for isRedAlliance in FMSUtils

**Review by tnrz135 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/greenblitz/robotName/commands/swerve/MoveToClimbPosition.java (hunk 1 above)
Comment (greenblitz4590) [reply src/main/java/edu/greenblitz/robotName/commands/swerve/MoveToClimbPosition.java]: > I believe there is a function for isRedAlliance in FMSUtils
download

File: src/main/java/edu/greenblitz/robotName/commands/swerve/MoveToAmp.java [hunk 2]
```diff
@@ -6,6 +6,6 @@
 public class MoveToAmp extends MoveToPosition {
 
     public MoveToAmp() {
-        super(Field.ScoringPositions.AMP_SCORE_POSITION);
```
Comment (hillelv) [inline src/main/java/edu/greenblitz/robotName/commands/swerve/MoveToAmp.java:9]: there is only PRE_AMP in real scoring positions

**Review by yonnbro — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/Field.java [hunk 3]
```diff
@@ -97,10 +97,16 @@ public static class FieldConstants{
     }
 
     public static class ScoringPositions{
-        public final static Pose2d SPEAKER_SCORE_POSITION = new Pose2d(new Translation2d(14.140,5.85),new Rotation2d());
-        public final static Pose2d AMP_SCORE_POSITION = new Pose2d(new Translation2d(14.85,7.5),new Rotation2d());
-        public final static Pose2d CLIMB_POSITION = new Pose2d(new Translation2d(12.733,3.602),new Rotation2d());
-        public final static Pose2d PRE_AMP_SCORE_POSITION = new Pose2d(new Translation2d(14.85,7.3),new Rotation2d());
+        public final static Pose2d RED_SPEAKER_SCORE_POSITION = new Pose2d(new Translation2d(14.140,5.85),new Rotation2d());
+        public final static Pose2d RED_SCORE_POSITION = new Pose2d(new Translation2d(14.85,7.5),new Rotation2d());
+        public final static Pose2d RED_CLIMB_POSITION = new Pose2d(new Translation2d(12.733,3.602),new Rotation2d());
```
Comment (roygold8) [inline src/main/java/edu/greenblitz/robotName/Field.java:102]: There are 3 positions :(

**Review by roygold8 — APPROVED** [review]

**Pull request merged**